third_party/blink/renderer/platform/wtf/hash_table.h

```diff
@@ -357,15 +358,16 @@ class HashTableConstIterator final {
 
   GetType Get() const {
     CheckModifications();
-    return position_;
+    return &container_->table_[*position_];
   }
   typename Traits::IteratorConstReferenceType operator*() const {
     return Traits::GetToReferenceConstConversion(Get());
   }
   GetType operator->() const { return Get(); }
 
   const_iterator& operator++() {
-    DCHECK_NE(position_, end_position_);
+    assert(position_ != end_position_);
```

Comment:
I think, it would be better to either use `CHECK_NE` or we could create a new set of `REPLAY_CHECK` `define`s which simply point to their `CHECK` counterparts to avoid unnecessary code changes.